core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -834,6 +834,17 @@ public ConditionalWriter createConditionalWriter(String tableName) throws TableN
     return createConditionalWriter(tableName, null);
   }
 
+  public ConditionalWriter createConditionalWriterForFateTable(String tableName)
+      throws TableNotFoundException {
+    ConditionalWriterImpl writer = (ConditionalWriterImpl) createConditionalWriter(tableName);
```

Comment:
Should use the ConditionalWriterConfig() object to set the number of threads, like the following.

```java
    AccumuloClient client = ...;
    var cwConfig = new ConditionalWriterConfig().setMaxWriteThreads(maxThreads);
    client.createConditionalWriter(table, cwConfig);
```


Why add this method to client context instead of somewhere in the fate code?

Reply:
Ah missed the createConditionalWriter method that takes a config

Was added here since it was accessing a private method. Since I removed the client property and replaced with a property, access to this method wasn't needed anymore so moved the method to `UserFateStore`